sample-queries.csv

```diff
@@ -545,48 +545,45 @@ Notifications (beta),POST,create a visual notification,/beta/me/notifications,Pl
   ""displayTimeToLive"": ""60""
 }",Content-Type: application/json
 Search(beta),POST,search message,/beta/search/query,enable Mail.Read,https://aka.ms/mssearchapi,AAD,"{
```

Comment:
Add a space between Search and Beta : Search (beta)
Same on all the other requests